src/init/AppRunner.ts

```diff
@@ -112,19 +108,34 @@ export class AppRunner {
   }
 
   /**
-   * Resolves a path relative to the current working directory,
-   * falling back to a path relative to this module.
+   * Creates the main app object to start the server from a given config.
+   * @param loaderProperties - Components.js loader properties.
+   * @param configFile - Path to a Components.js config file.
+   * @param variables - Variables to pass into the config file.
    */
-  protected resolveFilePath(cwdPath?: string | null, modulePath = ''): string {
-    return typeof cwdPath === 'string' ?
-      absoluteFilePath(cwdPath) :
-      joinFilePath(__dirname, '../../', modulePath);
+  protected async createApp(
```

Comment:
This should be public then. Main goal of that function was in case someone wanted to be able to manually start and stop a server in their project.